mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -662,6 +662,15 @@ getTuningProblemStr(RockGemmGemmWrapperInterface gemmGemmOp,
   else
     problemOS << "false" << sep;
 
+  if (isAttention) {
+    auto attentionOp = cast<AttentionOp>(gemmGemmOp);
+    problemOS << "-causal ";
+    if (attentionOp.getCausal())
+      problemOS << "true" << sep;
+    else
+      problemOS << "false" << sep;
```

Comment:
Does this mean, it is changing problem config string for the attention ops ? 
Would previous attention problem configs continue working in backward compatible way ?

Reply:
> do they parse it or just treat it as a string? 

They just use it as string for compilation cache purposes. ProblemConfig is the Key and perfConfig is the value in compilation cache hashMap. 

It should be okay i think.